tests/end_to_end_integration/.gitignore

```diff
@@ -1,2 +1,4 @@
 config.yaml
 kustomization.yaml
+training-data-config-compiled.yaml
+validation-data-config-compiled.yaml
```

Comment:
nit: We normally name our files with underscores and argo variables with dashes. You may want to refactor these to use underscores.

Reply:
for better or worse, that has not been the standard practice for config/argo yamls. E.g. see the workflows/argo directory or the various config yamls in vcm-workflow-control. I'm going to leave this as is.